jbs-ubi8-builder/Dockerfile

```diff
@@ -202,7 +181,7 @@ LABEL \
     maintainer="Stuart Douglas <sdouglas@redhat.com>"
 
 # Clear package manager metadata
-RUN microdnf clean all && [ ! -d /var/cache/yum ] || rm -rf /var/cache/yum && mkdir /project && chown 185:185 /project
+RUN rm -rf "/var/cache/yum" "/var/lib/rpm" "/var/lib/dnf" && mkdir /project && chown 185:185 /project
```

Comment:
microdnf clean removes _some_ files from the cache/lib directories but not everything. Therefore its more efficient and faster just to remove everything.